pravega-client-examples/src/test/java/io/pravega/example/streamprocessing/WorkerProcessConfig.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package io.pravega.example.streamprocessing;
+
+import io.pravega.client.ClientConfig;
+import lombok.Builder;
+
+@Builder
```

Comment:
This is using lombok.

Reply:
Yes, I now remember that I used lombok in quite a few places. I have now updated some of the try-with-resource blocks with lombok @Cleanup. There are still a few places that I left unchanged because I didn't want to change the behavior.